### ISSUE TITLE
[华农兄弟] 华农兄弟看望一下兄弟顺便挖棵黄花倒水莲回去煲汤很香哦 - https://youtu.be/DSk6GGV5_04

### ISSUE THREAD

**Issue by immoonancient** [immoonancient/YTSubtitles#1531]
https://youtu.be/DSk6GGV5_04
<p>&#35270;&#39057;&#21457;&#24067;&#20110; September 30, 2020 at 06:30PM</p>
<p><small><i>&#30001; <a href="https://ifttt.com/applets/104837875d">IFTTT</a> &#33258;&#21160;&#21457;&#24067;</i></small></p>

**Comment by vincenthychow** [comment]
bot, please upload
#https://youtu.be/DSk6GGV5_04

#华农兄弟：看望一下兄弟，顺便挖棵黄花倒水莲回去煲汤，很香哦

#Huanong Brothers: We visit our buddy, then dig a milkwort back home and make a chicken soup.Nice!
#Huanong Brothers: We visit our buddy, then dig a milkwort back home and make a chicken soup. It smells so good!


0:00:00.000,0:00:03.769
# 兄弟的兄弟哦 那麼歡迎我是不是  嘿
My buddy’s buddy. She’s so excited to see me, right? Heh

0:00:04.246,0:00:06.435
# 別咬我啊  聽話  趴下
Don’t bite me! Lie down, be a good girl!

0:00:07.163,0:00:08.562
# 趴下  別動
Lie down! Freeze!

0:00:13.456,0:00:16.228
# 哈哈哈  太嚇人了
Hahaha, it’s too scary!

0:00:21.833,0:00:23.026
# 聽話  趴下
Lie down. Be a good girl!

0:00:28.089,0:00:30.673
# 你看 這麼兇 這個是母的哦
Look! She’s so fierce. This is a girl, you know.

0:00:36.640,0:00:38.710
# 你看 秋天  這個花都開啦
Look! It’s autumn now and the flowers have already blossomed.

0:00:39.808,0:00:41.328
# 這是五倍子花
These are the flowers of nutgalls.

0:00:44.179,0:00:45.733
# 好香啊 淡淡的 香味
It smells so good. The fragrance, a subtle one.

0:00:47.333,0:00:48.234
# 一樹都是
They’re all over the tree.

0:00:55.611,0:00:56.787
# 你看  這個是
Look! This is…

0:00:57.313,0:00:59.176
# 我們這邊叫甚麼  金ang子
How do we call it here? “Jin ang zi”

0:01:00.211,0:01:01.283
# 它這個表面的刺
They can be used to make infused liquor,

0:01:02.063,0:01:04.269
# 把它搞掉然後用來 泡酒也可以
after removing the spikes on its surface.

0:01:05.380,0:01:08.513
# 那個  直接吃也可以  但是要把裡面的籽要去掉哦
Actually, it can be eaten right away, in fact. But the seeds inside need to be removed first.

0:01:09.899,0:01:12.328
# 今年的果  今年的野果都是很
The fruits this yere, the wild fruits this year are really,

0:01:12.761,0:01:15.000
# 很多的哦  今年都是豐收年
really plentiful, you know! This is a year of Bumper Yield.

0:01:45.222,0:01:46.741
# 你看  這還有bi子(杮子)哦
Look! Here are also “bi zi” (persimmon),

0:01:47.546,0:01:48.378
# 小bi子(杮子)
tiny “bi zi”.

0:01:49.068,0:01:51.583
# 不是很大的那種  很是很甜的那種
These are not that big, but are really sweet.

0:01:51.840,0:01:54.000
# 你看  這裡有一棵個黃花倒水蓮哦
Look! Here is a milkwort (transliteration: Huanghua daoshuilian).

0:01:54.398,0:01:55.745
# 這個可以用來煲雞湯的
This can used to make a chicken soup.

0:01:56.215,0:01:57.888
# 它的根可以用來煲雞湯的
Its root can be used to make a chicken soup.

0:01:58.396,0:01:59.194
# 特別好吃
It’s really tasty.

0:01:59.481,0:02:00.416
# 我去借個鋤頭來
Let me borrow a hoe,

0:02:01.354,0:02:02.358
# 把它挖回家
and dig it back home.

0:02:02.828,0:02:04.743
# 你看  這上還有小黃梨
Look! Here are also some “little brown pears” (a kind of wild fruits) up there.

0:02:05.797,0:02:09.789
# 搞一點來吃  這個應該  要煮熟了才可以吃的這個
Let me pick some. I think, they need to be cooked first, before they can be eaten.

0:02:10.545,0:02:11.618
# 很澀的哦
It has a strong astringent taste, you know.

0:02:14.560,0:02:15.376
# 好多啊
Really a lot!

0:02:16.210,0:02:17.025
# 一串都是
A whole string of them

0:02:23.360,0:02:25.345
# 你看  這就是  像不像小梨
Look! Here they are. Do they look like little pears?

0:02:25.895,0:02:27.981
# 我們這野生的小黃梨就這樣子的
These are the wild “little brown pears” here at our place.

0:02:30.706,0:02:32.414
# 嗯  好澀的這個
Um! It's really astringent!

0:02:35.225,0:02:36.418
# 甜甜的澀澀的
It’s a bit sweet and astringent.

0:02:37.211,0:02:39.485
# 不是很  不是澀澀  是很澀
It’s not very... not a bit astringent, but pretty astringent.

0:02:44.686,0:02:47.184
# 以前還小的時候  用那個水煮一下
We used to blanche them in water when we were small.

0:02:47.864,0:02:49.452
# 也可以吃的  當零食吃
They can also be, eaten as snacks.

0:02:49.847,0:02:51.246
# 煮了之後它就沒那麼澀啦
They’re less astringent after being blanched.

0:02:51.970,0:02:53.815
# 就一個個可以當零食吃就這樣
Then they can be eaten as snacks. Eat the whole thing, see?

0:02:55.192,0:02:56.076
# 吃完就可以了
Just eat the whole thing.

0:02:59.602,0:03:02.151
# 剛剛吃了那個黃梨啊  好澀啊
I’ve just eaten the “little brown pears”. So astringent!

0:03:03.585,0:03:05.380
# 搞個糖吃一下  中和一下
To neutralise it, let me get some honey.

0:03:10.453,0:03:13.329
# 這個多糖啦  這個都有一斤那麼多
Here’s a lot of honey. It’s around half a kilogram. So much!

0:03:20.320,0:03:23.551
# 我去把個  那個 那個根挖出來
Let me go and then, you know, I mean, dig the root out.

0:03:29.849,0:03:33.240
# 今天沒穿長褲  穿個短褲過來
I am not wearing trousers today. I’m now in my pants.

0:03:34.552,0:03:36.586
# 但是  不影響哦
But, it doesn’t matter.

0:03:37.433,0:03:38.522
# 一樣可以挖回家
I can still dig it back home.

0:03:42.450,0:03:43.523
# 看下兄弟家
Let me see if my buddy,

0:03:44.314,0:03:46.297
# 看下  看下兄弟家有沒有雞
see, see if my buddy has any chickens here.

0:03:47.073,0:03:47.923
# 搞隻雞來
Let me fetch a chicken.

0:03:49.192,0:03:50.042
# 才叫完美
It would be perfect!

0:03:53.074,0:03:55.211
# 兄弟家的這個黃花倒水蓮
Buddy’s, buddy’s milkworts,

0:03:56.063,0:03:57.497
# 再配上兄弟家的雞
and buddy’s chicken,

0:03:58.658,0:03:59.902
# 加上我家的鹽
together with my salt,

0:04:01.271,0:04:03.151
# 絕對是絕配
is absolutely a perfect combination.

0:04:05.664,0:04:07.732
# 你看  夠大吧
Look! It’s big, isn’t it?

0:04:25.854,0:04:27.476
# 這個樹枝不要的這個
I don’t need the, the branches here.

0:04:42.437,0:04:45.072
# 好啦  要把它洗乾淨來
Alright! Let me wash it.

0:04:46.613,0:04:47.720
# 去溝裡面洗一下
Let me wash it in the stream.

0:04:51.805,0:04:53.839
# 你看 搞定
Look! It’s done!

0:04:54.655,0:04:57.000
# 就用這個根  來煲湯
I’m gonna just use this root, to make a soup.

0:04:57.823,0:04:59.359
# 其它的就不要 只要根
I'll throw away the rest, keeping only the root.

0:05:00.196,0:05:00.788
# 好啦
Alright!

0:05:03.361,0:05:04.725
# 先去洗一下  清理一下 503
Let me wash it, clean it first.

0:05:08.886,0:05:10.817
# 哎呀  好舒服啊
Aya! It’s so great.

0:05:11.481,0:05:13.378
# 這裡用山泉水洗一下腳吧
Let me wash my feet in the spring water first.

0:05:14.006,0:05:14.821
# 舒服
Wonderful!

0:05:16.908,0:05:19.887
# 你看  這樣洗一下   洗完就很漂亮啦
Look! It would be really pretty, after being washed.

0:05:23.860,0:05:25.242
# 這個要曬乾來的哦
It needs to be dried under the sun first, you know.

0:05:25.813,0:05:27.000
# 不是說直接煲的
It can’t be used right away.

0:05:30.657,0:05:34.615
# 兄弟一臉不情願  他說挖了好多  沒事
My buddy looks very upset. He said that I’ve taken too much. No worries. I'm gonna,

0:05:35.227,0:05:37.640
# 到時他  叫他一起吃   一起來吃就好啦
I’m gonna, ask him to enjoy, enjoy with us. He would be fine.

0:06:11.885,0:06:13.559
# 山泉水  最舒服
Spring water, is always the best.

0:06:18.492,0:06:21.609
# 好啦  就搞了那麼一點點  一大棵
Alright! So just a tiny little bit here, from such a big plant.

0:06:22.265,0:06:24.728
# 可以啦  收工收工
Alright! Let’s go, let’s go.

0:06:27.584,0:06:29.241
# 回去 曬乾來
Let’s go home, dry them,

0:06:29.589,0:06:30.696
# 搞隻雞來燉
fetch a chicken, and stew everything.

0:06:32.210,0:06:33.780
# 這個味道絕對OK
The taste would be absolutely OK.

0:06:34.265,0:06:36.128
# 可以啦 就這一捆
Alright! Just a bunch here.

0:06:38.327,0:06:39.503
# 總算沒有白來
At least this trip is worthy.

0:06:42.280,0:06:44.504
# 等一下我帶回去  看下兄弟在幹嘛
I’ll take it back in a while. Let’s see what my buddy is doing.

0:06:57.691,0:07:00.567
# 我也會  我朼會搖啦  我也會割啦
Now I also, I can also rip the honey. I can harvest honey now.

0:07:12.188,0:07:13.329
# 全都是蜜
It’s filled with honey.

0:07:19.395,0:07:24.326
# 割  割這個糖也要有技術的哦  沒有技術割不了
Har, harvesting honey requires skills, you know. Without the skills, it’s impossible to do it right.

0:07:26.240,0:07:28.784
# 好多啊  全往下留[SIC]啦
A lot! It’s all flowing down.

0:07:52.805,0:07:55.698
# 你看  這個  晶瑩剔透
Look! It’s, crystal clear.

0:07:58.608,0:08:03.218
# 哇  這個過濾網  這個濾網過完之後就那個
Wa! This filtering net, after being filtered with this net, it would be, you know,

0:08:04.638,0:08:05.917
# 就很漂亮的蜜啦
the honey would be really pretty.

0:08:25.859,0:08:28.666
# 你看這個過濾網  過了之後就很乾淨啦
Have a look at this filtering net. The honey would become very pretty after filtering.

0:08:39.100,0:08:40.464
# 來 乾杯
C’mon! Cheers!

0:08:43.780,0:08:44.715
# 啊~
Ah!

0:08:45.766,0:08:47.131
# 誒呀  好甜啊
Eya! It’s really sweet.

0:08:49.411,0:08:50.484
# 有那花粉的味道
There’s a scent of pollen.

0:08:55.808,0:08:56.726
# 啊~
Ah!

0:09:00.131,0:09:01.959
# 好啦好啦  不要淹在這裡  去採蜜
Alright, alright! Don’t get drown here. Go and collect the nectar.

0:09:04.890,0:09:06.907
# 這裡都差不多五十斤  很快呦
It’s already almost twenty five kilograms here. It doesn’t take long.

0:09:11.254,0:09:13.735
# 這個雞用那個恭花倒水蓮來燉了
These chickens are to be stewed with the milkwort.

0:09:14.722,0:09:16.155
# 這是蘆花雞來喲  這個
These are Luhua chickens, you know.

0:09:24.001,0:09:26.636
# 這個  這個是黃花倒水蓮
This is, this is the milkwort.

0:09:27.236,0:09:29.579
# 曬的不是很乾  本來要曬乾了味道更好
It’s not really that dry. It would taste much better if it's dried for a longer time, in fact.

0:09:30.291,0:09:31.999
# 現在逼不及待的
But now I can’t wait any longer.

0:09:33.358,0:09:35.478
# 先  先燉幾隻雞來
Let me, let me stew with a couple of chickens first,

0:09:35.599,0:09:36.431
# 試一下效果
and see if it’s good.

0:09:55.893,0:09:58.563
# 好  慢慢燉  要燉一個小時左右哇
Alright! It takes time. It needs to be stewed for around an hour,

0:09:59.022,0:10:00.764
# 才把那個雞的味道燉出來
so that the flavour of chickens can be released.

0:10:01.999,0:10:03.912
# 現在這樣生火  要等一下
That’s how we set the fire now. It needs a while.

0:10:05.016,0:10:06.741
# 等下它自然回一下火就好啦
It will glow itself. I don't need to take care of it.

0:10:07.444,0:10:09.069
# 從白天燉到晚上
From daylight to the evening

0:10:11.394,0:10:12.673
# 味道絕對OK
The taste is absolutely OK.

0:10:13.006,0:10:13.735
# 放點監
Put some salt.

0:10:28.112,0:10:29.923
# 別掉了哦  掉了就麻煩了
Don’t drop. It would be a disaster otherwise.

0:10:30.813,0:10:31.388
# 嘗一下
Let me try.

0:10:35.177,0:10:38.757
# 這個雞那個高  你看 它的腳你看  那麼高
This chicken is so tall. Look! Its foot, see? It’s so long.

0:10:40.917,0:10:43.964
# 我們家的小母雞都是小短腿  她的怎麼那高的腿
The feet of my little hens here are all tiny and short. Why are its feet so long?

0:10:44.357,0:10:45.000
# 大長腿
A big long foot

0:10:54.431,0:10:56.294
# 嗯  果然不錯哦
Um! It’s really not bad.

0:11:00.796,0:11:02.143
# 蘆花雞就是不于樣
Luhua chickens are really different.

0:11:03.280,0:11:04.215
# 肉質很緊
The meat is firm.

0:11:22.889,0:11:23.893
# 你們想不想吃
Do you want it?

0:11:24.513,0:11:25.013
# 哈
Hm?

0:11:25.961,0:11:26.673
# 想不想吃
Do you?

0:11:27.792,0:11:29.415
# 想吃要等一下啊  我吃完來
You gotta wait. Let me, finish first.

0:11:31.191,0:11:32.796
# 我吃肉  你們吃骨頭
Meat is mine, the bone is yours.

0:11:37.609,0:11:39.867
# 要好好帶崽 誒呦  你舔到沒有
Take good care of your children, alright? Eya! Did you lick it?

0:11:40.127,0:11:41.234
# 沒有  沒舔到
No, you didn’t.

0:11:52.469,0:11:53.353
# 給老母狗吃
It’s for the great mom.

0:11:55.607,0:11:57.555
# 老母狗最辛苦  老母狗
The great mom is always having a hard time, this great mom.

0:11:59.775,0:12:00.882
# 湯也好吃哦
The soup is also very tasty.

0:12:09.886,0:12:10.873
# 這樣搖一搖
Swaying it here,

0:12:11.537,0:12:12.696
# 很快就冷了的
can cool it down very quickly.

0:12:24.468,0:12:25.232
# 香味
Good scent!

0:12:26.200,0:12:28.972
# 啥味都沒有  就除了香味  就是香味
Nothing else, apart form good scent, there's still only good scent,

0:12:30.199,0:12:30.699
# 雞味
scent of chicken!

0:12:33.259,0:12:34.898
# 要嘗嘗  味道都沒有嘗出來
Let me try again. I haven’t tried enough.

0:12:35.501,0:12:36.728
# 搞雞抓吃一下
Let me get a chicken foot,

0:12:38.826,0:12:39.933
# 搞個雞翅吃一下
get a chicken wing.

0:12:57.520,0:12:59.194
# 你看  這種雞的骨頭好大的
Look! This kind of chickens have very big bones,

0:12:59.839,0:13:00.706
# 就是沒甚麼肉
but not too much meat.

0:13:02.269,0:13:04.046
# 這種雞適合煲湯特別好吃
They're good for making soups. It's exceptionally tasty.

0:13:09.164,0:13:10.271
# 連肥肉都沒有
There’s even hardly any fat.

0:13:12.274,0:13:14.222
# 好啦  今天的視頻就到這裡啦
Alright! The video ends here.

0:13:14.832,0:13:15.372
# 吃飯啦
It’s time for dinner now.

0:13:16.945,0:13:18.773
# 大家都等著吃這個  這鍋雞湯
Everybody is waiting, waiting for this chicken soup.

0:13:20.665,0:13:21.789
# 全都人都看著我吃
Everybody is watching me eating.

0:13:22.629,0:13:23.805
# 搞很我再也不好意思啦
It makes me feel very embarrassed.

**Comment by vincenthychow** [comment]
已發佈
